.travis.yml

```
@@ -22,8 +22,8 @@ branches:
   only:
   - master
   - development
-  - release/*
-  - hotfix/*
+  - release\/.*
+  - hotfix\/.*
```

I think in travis u need to encapsulate regex with two slashes:
```suggestion
  - /^release\/.*$/
  - /^hotfix\/.*$/
```

---

lets try it out